docs/advertising/ad-blocking.rst

```diff
@@ -0,0 +1,71 @@
+Ad blocking
+===========
+
+Ad blockers fulfill a legitimate need
+to mitigate the significant downsides of advertising
+from tracking across the internet,
+security implications of third-party code,
+and impacting the UX and performance of sites.
+
+At Read the Docs, we specifically didn't want those things.
+That's why we built the our :doc:`Ethical Ad initiative <ethical-advertising>`
+with only relevant, unobtrusive ads that respect your privacy
+and don't do creepy behavioral targeting.
+
+Advertising is the single largest source of funding for Read the Docs.
+To keep our operations sustainable, we ask that you either
+:ref:`allow our Ethical Ads <advertising/ad-blocking:Allowing Ethical Ads>`
+or :ref:`go ad-free <advertising/ad-blocking:Going ad-free>` by donating.
```

Comment:
Since we call them "supporters" in the database and in the site footer, let's use the phrase "becoming a supporter". Since we call Gold "becoming a gold member" I don't want to have to differentiate between a "Gold Member" and a "Supporting Member" or a "Contributing Member".

Let's just use "Gold Member" and "Supporter".